x/consensus/types/msgs.go

```diff
@@ -16,7 +16,7 @@ func (msg MsgUpdateParams) ToProtoConsensusParams() (cmtproto.ConsensusParams, e
 		return cmtproto.ConsensusParams{}, errors.New("all parameters must be present")
 	}
 
-	if msg.Abci != nil && msg.Feature != nil && msg.Feature.VoteExtensionsEnableHeight != nil {
+	if msg.Feature != nil && msg.Feature.VoteExtensionsEnableHeight != nil {
```

Comment:
seems like abci should stay: https://github.com/cosmos/cosmos-sdk/pull/20347/files#diff-d6f7bad5e1ef27d60e3617dff82c14a3b6a992b2aac209838d8a8734f9b6cd79R16-R19